src/Particular.ThroughputCollector/ThroughputCollectorHostBuilderExtensions.cs

```diff
@@ -51,20 +51,9 @@ public static IHostApplicationBuilder AddThroughputCollector(this IHostApplicati
         return hostBuilder;
     }
 
-    public static IHostApplicationBuilder AddThroughputCollectorPersistence(this IHostApplicationBuilder hostBuilder, string persistenceType, string? errorQueue = null, string? serviceControlQueue = null)
+    public static IHostApplicationBuilder AddThroughputCollectorPersistence(this IHostApplicationBuilder hostBuilder, string persistenceType)
```

Comment:
do we need this at all?

Reply:
It is called from the ThroughputComponent in ServiceControl